shared_kv_storage/db_store/packages/db_store_wrapper.h

```diff
@@ -207,6 +207,51 @@ namespace db_store {
         const ByteArray& inId,
         const ByteArray& inValue);
 
+    // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    /**
+     * Updates a key->value pair in the database store. Appends/prepends a
+     * new string to the value separated by comma. The beahaviour is similar
+     * to pushing to the head/tail of a deque. Adds key-value pair if no
+     * entry is found.
+     * Primary expected use: python / untrusted side
+     *
+     * @param table     table name
+     * @param inId      id byte array
+     * @param inValue   data to be appended/written
+     *
+     * @return
+     *  TCF_SUCCESS  id->value updated/stored
+     *  else         failed, data store unchanged
+     */
+    tcf_err_t db_store_csv_extend(
```

Comment:
You can keep csv apis in this wrapper.

Reply:
Can we move csv API's to to a separate file. like `db_store_csv_wrapper.h`